<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostMonster</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="EditForm" name="editPage"/>
      <widget class="RecordForm" name="recordPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRecord"/>
   <addaction name="separator"/>
   <addaction name="actionPlugins"/>
  </widget>
  <action name="actionPlugins">
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/star</normaloff>:/icons/star</iconset>
   </property>
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/diagram</normaloff>:/icons/diagram</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/record-dim</normaloff>:/icons/record-dim</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/file-open</normaloff>:/icons/file-open</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/file-save</normaloff>:/icons/file-save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/file-saveas</normaloff>:/icons/file-saveas</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="postmonster.qrc">
     <normaloff>:/icons/file-new</normaloff>:/icons/file-new</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RecordForm</class>
   <extends>QWidget</extends>
   <header>widgets/recordform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EditForm</class>
   <extends>QWidget</extends>
   <header>widgets/editform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="postmonster.qrc"/>
 </resources>
 <connections/>
</ui>
